test/performance/README.md

```diff
@@ -111,13 +111,88 @@ To generate the sample workload for KubeStellar performance experiments proceed
     - 10 config maps containing a 2048-character random string
 
 
-
-
 5. Clean up the generated workload Kubernetes API objects from your cluster:
 
    ```bash
    $ cd $CL2_DIR
    $ ./cleanup.sh
    ```
 
-### Collect the creation & update timestamps for workload benchmark (OPTIONAL) (TO DO: WORK-IN-PROGRESS)
\ No newline at end of file
+
+### Collect the creation & update timestamps for workload benchmark (OPTIONAL)
+
+
+To collect the creation and status_update timestamps for the benchmark workload objects proceed as following:  
+
+1. Run the metrics collection script:
+
+   a) cd into the `test/performance` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   $ cd .$HOME/kubestellar/test/performance
+   ```
+
```

Comment:
At this point it would be worth pointing out that the following depends on the dependencies listed in `requirements.txt`.

Reply:
Done!